kura/distrib/src/main/resources/common/gen_config_ini.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+TEMPLATE=$1
+ROOT=$2
+
+usage() {
+    >&2 echo "Usage: gen_config_ini.sh <config.ini template> <plugin root DIR_NAMEectory>"
```

Comment:
Probably a leftover from a string substitution

```suggestion
    >&2 echo "Usage: gen_config_ini.sh <config.ini template> <plugin root directory>"
```